storage/common.go

```diff
@@ -0,0 +1,68 @@
+package storage
```

Comment:
I'm unsure if the AccessPolicyXML is used in other places other than Tables and Blob (I'll have a look at Queues later once I'm back online). Agree that the common.go naming leaves a bit to be desired. I like the idea of storagepolicy.go as per @mcardosos 

Even if it does turn up queues use it, the name of storagepolicy.go is still valid and descriptive.

Any objections if I rename and push?